gwcli/client.py

```diff
@@ -422,7 +422,7 @@ def ui_command_disk(self, action='add', disk=None, size=None):
         in the configuration, the cli will attempt to create it for you.
 
         e.g.
-        disk add <disk_name> <size>
+        disk add <pool_name.disk_name> <size>
```

Comment:
Nit: wouldn't this be ```<pool_name.image_name>```? Otherwise, what's the difference w/ the line below?

Reply:
Yeah, I fixed this and remove below. Did a force respush so it should be fixed.